src/shared/settings.ts

```diff
@@ -544,6 +544,8 @@ const devSettings = {
     forceInstallTools: Boolean,
     telemetryEndpoint: String,
     telemetryUserPool: String,
+    reportRequestIds: Boolean,
+    renderDebugDetails: Boolean,
```

Comment:
why make renderDebugDetails optional? is it potentially noisy?

Reply:
Yeah it makes the connection picker a bit noisy. Hiding it under a dev setting for now